oshi-core/src/test/java/oshi/util/ExecutingCommandTest.java

```diff
@@ -30,7 +30,7 @@
  */
 public class ExecutingCommandTest {
 
-    private static final String ECHO = "echo Test";
+    private static final String ECHO = System.getProperty("os.name").startsWith("Windows") ? "cmd.exe /C echo Test" : "echo Test";
```

Comment:
You might consider using `SystemInfo.getCurrentPlatformEnum().equals(PlatformEnum.WINDOWS)` as the conditional here.  Ultimately it will do the same thing, so it's mostly a matter of style whether to use an enum that's already checked the `os.name` property.